src/MatLang.jl

```diff
@@ -1,20 +1,76 @@
 module MatLang
 using Requires
+using LinearAlgebra
 
-export clcM
+# Main Functions
+export clcM, eyeM, diagM
+
+# Native Julia functions that are just a copy:
+export zerosM, onesM, randM
+
+
+# Command Window (REPL console):
 ################################################################
 """
     clcM
 
 Clear Command Window for Juno or Atom IDE
 """
 function clcM()
-#    Juno.clearconsole();
+#    Juno.clearconsole(); # defined in @require instead
 end
 ################################################################
 
 
 ################################################################
+
+# Matrices and Arrays:
+################################################################
+const zerosM = zeros
+################################################################
+const onesM = ones
+################################################################
+const randM = rand
+################################################################
+"""
+    eyeM(s1,s2,...)
+    eyeM(sizeAsTuple)
+
+Creates Identity matrix (can be non-square matrix).
+Example
+'''jl
+mEye=eyeM(2,3); # [1 0 0; 0 1 0]
+mEye2=eyeM((2,2))  # giving size as a Tuple. In Julia we use (2,2) instead of giving it as [2,2]
+s1=size(ones(2,3))
+mEye3=eyeM(s1)  # giving size as a variable (Tuple).
+'''
+"""
+function eyeM(args...)
```

Comment:
```julia
eyeM(T::Type=Int64, first_idx::Integer, indices::Interge...) = Matrix{T}(I, first_idx, indices...)
eyeM(T::Type=Int64, indices::Tuple) = eyeM(T, indices...)
```

Reply:
Could you explain this a little?

Reply:
This splits the dispatching rule (similar to the design of classes in other languages) and core-implementation.

You can check https://docs.julialang.org/en/v1/manual/methods/#Iterated-dispatch-1 for the explanation.

Reply:
OK. That's cool. I will use this in the next commit.

Reply:
This syntax does not work. 

I want to try something simpler like this but it doesn't work:
eyeM(T::Type=Int64, indices::Integer...)=Matrix{T}(I, indices...)
eyeM(a::Array,T::Type=Int64)=Matrix{T}(I, Tuple(a))
We should have both optional positional arguments and variable number input arguments at the end! I don't know how to make this work.

Reply:
If eyeM was a type we could try this:
```julia
eyeM{T}(indices::Integer...)=Matrix{T}(I,indices...)
eyeM{T}(ia::Array)=Matrix{T}(I,Tuple(a))
```